app/scripts/scrollHelp.js

```diff
@@ -1,14 +1,18 @@
-const overlay = `
+const { ipcRenderer } = require('electron');
+
```

Comment:
Dit is een PR voor auto update. Dit van scrollhelp zou er niet moeten inzitten denk ik? ;)

Reply:
Ik had een fout gemaakt van branch als ik iets had aangepast aan scroll help. Zal er op letten volgende keer.